addons/common/XEH_preInit.sqf

```diff
@@ -6,5 +6,23 @@ GVAR(aceSafemode) = IS_MOD_LOADED(ace_safemode);
 GVAR(aceFastroping) = IS_MOD_LOADED(ace_fastroping);
 GVAR(aceFatigue) = IS_MOD_LOADED(ace_advanced_fatigue);
 GVAR(aceMedical) = IS_MOD_LOADED(ace_medical_engine);
+GVAR(aceTagging) = IS_MOD_LOADED(ace_medical_engine);
```

Comment:
I don't think it's correct. 😛 At least once my review has found something wrong. 🎉 
```suggestion
GVAR(aceTagging) = IS_MOD_LOADED(ace_tagging);
```